_test_resources/index.ts

```diff
@@ -63,3 +63,197 @@ vpc.subnets.apply((subnets) => {
         securityGroupId: sg.id,
     })
 })
+
+
+// RDS Resources
+
+const username = "testuser"
+const password = "testpassword"
+
+
+const subnetGroup = new aws.rds.SubnetGroup('rds-subnet-group', {
+    subnetIds: vpc.privateSubnetIds,
+    tags: {
+        Name: 'Klotho DB subnet group',
```

Comment:
minor: would it be more helpful if the tag name were something like "Klotho", and the value were something like the test run (or setup id, etc)? Basically, something where if we have multiple of these in one region, the tag value is unique in a way that lets us figure out which one we want to keep during a cleanup?

Reply:
yeah i do think tags are likely a good thing to add. Im going to push this since its already a large pr and then follow up with that